scoring.test.js

```diff
@@ -0,0 +1,125 @@
+const calculateScore = require('./scoring')
+
+describe('calculateScore', () => {
```

Comment:
Nothing to say here really - it's a very clear and comprehensive set of tests, laid out nicely